src/recipe.cc

```diff
@@ -29,4 +29,16 @@ std::vector<ShaderInfo> Recipe::GetShaderInfo() const {
   return impl_->GetShaderInfo();
 }
 
+std::vector<std::string> Recipe::GetRequiredFeatures() const {
+  if (!impl_)
+    return std::vector<std::string>();
+  return impl_->GetRequiredFeatures();
```

Comment:
return impl_ ? impl_->GetRequiredFeatures() : {};

Reply:
`{}` has compiler errors and done with `std::vector<std::string>()`.